lib/galaxy/webapps/galaxy/api/__init__.py

```diff
@@ -85,3 +86,7 @@ def get_admin_user(trans: SessionRequestContext = Depends(get_trans)):
     if not trans.user_is_admin:
         raise AdminRequiredException(require_admin_message(trans.app.config, trans.user))
     return trans.user
+
+
+def get_tours_registry(app: UniverseApplication = Depends(get_app)) -> ToursRegistry:
```

Comment:
Not sure, maybe we should move this function to `lib/galaxy/webapps/galaxy/api/tours.py`?. It seems probable that it will be used only in the `tours` route.

Reply:
Done. I didn't include the `UniverseApplication` annotation to avoid importing it into `api/tours.py` (it's already annotated in `get_app`)